charts/substra-backend/templates/deployment-server.yaml

```diff
@@ -222,33 +222,13 @@ spec:
           - name: statics
             mountPath: /usr/src/app/backend/statics
       {{- include "common.waitPostgresqlInitContainer" . | nindent 6 }}
-      - name: init-migrate
-        image: {{ include "substra-backend.images.name" (dict "img" .Values.server.image "defaultTag" $.Chart.AppVersion) }}
-        command: ['python', 'manage.py', 'migrate']
-        securityContext:
-          allowPrivilegeEscalation: false
-          runAsNonRoot: true
-          seccompProfile:
-            type: RuntimeDefault
-          capabilities:
-            drop: ["ALL"]
-        envFrom:
-          - configMapRef:
-              name: {{ include "substra.fullname" . }}-orchestrator
-          - secretRef:
-              name: {{ include "substra.fullname" . }}-objectstore
-          - configMapRef:
-              name: {{ include "substra.fullname" . }}-database
-          - configMapRef:
-              name: {{ include "substra.fullname" . }}-settings
-          - secretRef:
-              name: {{ include "substra-backend.database.secret-name" . }}
-        env:
-        - name: DJANGO_SETTINGS_MODULE
-          value: backend.settings.{{ .Values.settings }}
+      {{- include "common.waitInitMigrationsInitContainer" . | nindent 6}}
       {{- include "common.waitMinIOContainer" . | nindent 6 }}
       {{- include "common.waitRedisInitContainer" . | nindent 6 }}
       volumes:
+      - name: volume-wait-init-migrations
+        configMap:
+          name: {{ include "substra.fullname" . }}-wait-init-migrations
```

Comment:
Can you provide details on when is the `wait-init-migration.sh` script actually called?

Reply:
on the line 225 (`{{- include "common.waitInitMigrationsInitContainer" . | nindent 6}}`)